lib/fastlane/plugin/android_sdk_update/actions/android_sdk_update_action.rb

```diff
@@ -41,11 +41,22 @@ def self.run(params)
           Actions.sh "echo y | android update sdk --no-ui --all --filter tools"
         end
 
+        # Accept licenses for all available packages
+        FastlaneCore::CommandExecutor.execute(command: "echo y | #{sdk_manager} --licenses",
+                                              print_all: true,
+                                              print_command: false)
+
+        # Install packages
         packages.each { |package| UI.message("• #{package}") }
         FastlaneCore::CommandExecutor.execute(command: "echo y | #{sdk_manager} '#{packages.join("' '")}'",
                                               print_all: true,
                                               print_command: false)
 
+        # Ensure all installed packages are updated
+        FastlaneCore::CommandExecutor.execute(command: "echo y | #{sdk_manager} --update",
```

Comment:
In our case, we only want to update the packages we really need.
Otherwise it could take a while (especially for system-images)

Maybe as an option?!

Reply:
sounds good -- I'll adjust and update the branch